File: Julia/src/fizzbuzz.jl
function fizzbuzz(n)
    for i in 1:n
        if i % 15 == 0
            println("fizzbuzz")
        elseif i % 3 == 0
            println("fizz")
        elseif i % 5 == 0
            println("buzz")
        else
            println(i)
        end
    end
end


fizzbuzz(30)

File: Elixir/src/fizzbuzz.exs
fb = fn
    0, 0, _ -> "FizzBuzz"
    0, _, _ ->  "Fizz"
    _, 0, _ ->  "Buzz"
    _, _, x ->  x
end

fizzbuzz = fn n -> fb.(rem(n, 3), rem(n, 5), n) end

Enum.each 1..100, fn n -> IO.puts fizzbuzz.(n) end


